src/segment_owd_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOWDStringPlayingSong(t *testing.T) {
```

Comment:
I don't think the function name is correct

Reply:
That's a whoopsie 😅. Removed in c547971